services/wordpress/wordpress-rating.service.js

```diff
@@ -67,6 +54,22 @@ function RatingForExtensionType(extensionType) {
         },
       ]
     }
+
+    static render({ rating, numRatings }) {
+      const scaled = (rating / 100) * 5
+      return {
+        message: `${scaled.toFixed(1)}/5 (${metric(numRatings)})`,
+        color: floorCount(scaled, 2, 3, 4),
```

Comment:
It looks like the legacy service used the fixed/rounded scaled value in the color scale. I think that would technically mean there's a handful of boundary cases with rating values _really_ close to the scale steps that would end up colored differently.

For example if `scaled` has a value of `2.99` then I believe that would be yellowgreen, but in the legacy service (due to the `.toFixed` that same badge would have been green.

Reply:
Sure, it seems reasonable to use the rounded display value for the color.